### DISCUSSION TITLE
Is there a simple pointer locking OP ?

### DISCUSSION THREAD

**Discussion by cristianvogel** [cables-gl/cables_docs#664] [Answers & Questions] [answered]
I am designing some dials and faders, and need pointerLocking API in Cables. I saw that WASD has this feature, but wanted to know if it was around on its own, so I don't have to decipher the complex camera code in the WASD camera.

**Comment by pandrr** [comment, marked as answer]
yep was missing, i made an op here:  https://dev.cables.gl/edit/ds6IV2
